DP03_05_Using_User-Supplied_Catalogs.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<p>1) Ocasionally -&gt; Occasionally</p><p>2) I think we are supposed to avoid writing in the first person (or at least that is how the current tutorial notebook guidelines are written) so maybe rephrase "you could run" -&gt; the following error message may occur. Then later "lets get an instance" -&gt; "get an instance"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>